src/SFA.DAS.ASK.Application.UnitTests/Handlers/GetNonDfeOrganisations/GetNonDfeOrganisationsTestBase.cs

```diff
@@ -0,0 +1,54 @@
+﻿using MediatR;
+using NSubstitute;
+using NUnit.Framework;
+using SFA.DAS.ASK.Application.Handlers.RequestSupport.GetNonDfeOrganisations;
+using SFA.DAS.ASK.Application.Services.ReferenceData;
+using SFA.DAS.ASK.Application.Services.Session;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ASK.Application.UnitTests.Handlers.GetNonDfeOrganisations
+{
+    [TestFixture]
+    public class GetNonDfeOrganisationsTestBase
+    {
+
+
+        public GetNonDfeOrganisationsHandler Handler { get; set; }
+
+        public ISessionService sessionService { get; set; }
+        public IReferenceDataApiClient referenceDataApi { get; set; }
+
+        private Guid requestId = Guid.Parse("63be476e-0593-40c5-9b8d-8f0358a4d195");
```

Comment:
If these are fields that are accessed by the inherited classes, they should be `protected` rather than public and capitalised.  `protected ISessionService SessionService`
And they only need to be fields rather than properties.